packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -420,9 +420,9 @@ export const Audio = () => {
     const newUri = source.uri
     if (currentUriRef.current !== newUri) {
       currentUriRef.current = newUri
-      const imageUrl = trackImageSource?.source[2].uri ?? DEFAULT_IMAGE_URL
+      const imageUrl = trackImageSource?.source?.[2]?.uri ?? DEFAULT_IMAGE_URL
       const nextImageUrl =
-        nextTrackImageSource?.source[2].uri ?? DEFAULT_IMAGE_URL
+        nextTrackImageSource?.source?.[2]?.uri ?? DEFAULT_IMAGE_URL
```

Comment:
do we get artwork on the lock screen when offline?

Reply:
I still need to test that. We did, but haven't tried on device since the player change. Sim isn't showing me lock screen controls